### PR TITLE
Hardtacks, for pirates, sea-farers and landlubbers alike.

### PR DESCRIPTION
**Pull request by SharkeeByond**
## About The Pull Request

Ahoy! This PR adds a small fluff item, hardtacks.

## Why It's Good For The Game

This PR adds another NSV-specific item, ment for the pirates. You can also use it to beat people to death!

## Testing Photographs and Procedure

Ensure you have 2 containers with atleast 10 units of flour and water nearby, Do NOT mix them, this will create flour instead!
Open the crafting table and go to "Breads", on the bottom of the list will be "Hardtack"
Craft, it should appear below your character.

<details>
<summary>Screenshots&Videos</summary>

image
image

</details>

## Changelog
:cl:
add: Added hardtacks!
imageadd: Hardtack sprite
code: Hardtack code and recipe
/:cl:

### CODE REVIEW

**Review by Karmic-Skink — CHANGES_REQUESTED** [review]
Please move these all to the custom files in the NSV13 folder.

**Comment by SharkeeByond** [comment]
> Please move these all to the custom files in the NSV13 folder.

Should be done now

File: nsv13/code/modules/food_and_drinks/food/custom_food.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/obj/item/reagent_containers/food/snacks/hardtack //NSV13
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/food_and_drinks/food/custom_food.dm:1]: You don't need to keep the NSV tags when it's already in a custom file, and you need to include these files in the nsv13.dme file as well ("ticking" it in dream maker or VSC)

File: nsv13/code/modules/food_and_drinks/food/custom_food.dm [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	name = "hardtack"
+	desc = "Traditional pirate snack. Legally considered a dangerous weapon."
+	icon_state = "hardtack"
+	bonus_reagents = list(/datum/reagent/consumable/nutriment = 1)
+	list_reagents = list(/datum/reagent/consumable/nutriment = 2)
+	tastes = list("broken teeth" = 7,"depression" = 4,"flour and water" = 5)
+	foodtype = GRAIN
+	force = 9
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/food_and_drinks/food/custom_food.dm:9]: I suspect you wanted to use `throwforce` for this, or possibly just also add it.

File: nsv13/code/modules/food_and_drinks/food/custom_food.dm [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/obj/item/reagent_containers/food/snacks/hardtack //NSV13
+	name = "hardtack"
+	desc = "Traditional pirate snack. Legally considered a dangerous weapon."
+	icon_state = "hardtack"
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/food_and_drinks/food/custom_food.dm:4]:
```suggestion
	desc = "Traditional pirate snack. Legally considered a dangerous weapon."
	icon = 'nsv13/icons/obj/food.dmi'
	icon_state = "hardtack"
```
Need the custom icon *file* referenced here as well

File: nsv13/code/modules/food_and_drinks/recipes/tablecraft/recipes_custom [hunk 4]
```diff
@@ -0,0 +1,8 @@
+/datum/crafting_recipe/food/hardtack //NSV13
+	name = "Hardtack"
+	reqs = list(
+		/datum/reagent/water = 10,
+		/datum/reagent/consumable/flour = 10
+	)
+	result = /obj/item/reagent_containers/food/snacks/hardtack
+	subcategory = CAT_BREAD
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/food_and_drinks/recipes/tablecraft/recipes_custom:8]: This file has no extension, and ditto on the commenting and including in .dme

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

**Comment by SharkeeByond** [comment]
Closing this PR for now, too busy with irl stuff to continue. Those who are willing to start this PR again by themselves may use the sprites provided.

**Pull request closed without merging**